lib/tests/streamlit/forward_msg_queue_test.py

```diff
@@ -58,6 +58,7 @@
 
 class ForwardMsgQueueTest(unittest.TestCase):
     def test_simple_enqueue(self):
+        """Enqueue a single ForwardMsg."""
```

Comment:
There's a bunch of minor cleanup in this file: adding docstrings to tests, fixing parameter ordering in `assertEqual` calls, etc. (The file also has some significant test changes to account for the add_rows removal!)